api/src/main/java/edu/wpi/first/shuffleboard/api/components/WidgetPropertySheet.java

```diff
@@ -31,12 +28,10 @@
 public class WidgetPropertySheet extends PropertySheet {
```

Comment:
name is misleading now that we're using it for things that aren't widgets.

Reply:
Should be a separate PR. I'll open an issue for it